### ISSUE TITLE
Building toolchain does not work - build-tools;30.0.0-rc2 not found

### ISSUE THREAD

**Issue by reald** [bastibl/gnuradio-android#2]
```
git clone --depth=1 https://github.com/bastibl/gnuradio-android.git
cd gnuradio-android/docker
docker build -t gnuradio-android .
```

Results in:

```
 ---> Using cache
 ---> 41e813121662
Step 36/47 : RUN yes | ./sdkmanager --sdk_root=${ANDROID_HOME} "build-tools;30.0.0-rc2"
 ---> Running in d5f40b0bacde
Warning: Failed to find package build-tools;30.0.0-rc2                          
The command '/bin/sh -c yes | ./sdkmanager --sdk_root=${ANDROID_HOME} "build-tools;30.0.0-rc2"' returned a non-zero code: 1
```

**Comment by bastibl** [comment]
Any chances your network got disconnected or the disk with the image was running out of space?

**Comment by reald** [comment]
Disk is not full, network up. Tried another machine with same result. Both run Ubuntu 20.04 LTS.

**Comment by bastibl** [comment]
Thanks for raising the issue. I was able to reproduce it on my system. They released `build-tools,` 30.0.0, i.e., the `rc2` had to be deleted. I just pushed a fix.
